Index: gnome-mplayer-0.9.6/src/support.c
===================================================================
--- gnome-mplayer-0.9.6/src/support.c
+++ gnome-mplayer-0.9.6/src/support.c	2009-07-12 19:47:02.000000000 +0300
@@ -849,7 +849,7 @@
 
         if (error_msg != NULL) {
             dialog = gtk_message_dialog_new(NULL, GTK_DIALOG_DESTROY_WITH_PARENT, GTK_MESSAGE_ERROR,
-                                            GTK_BUTTONS_CLOSE, error_msg);
+                                            GTK_BUTTONS_CLOSE, "%s", error_msg);
             gtk_window_set_title(GTK_WINDOW(dialog), _("GNOME MPlayer Error"));
             gtk_dialog_run(GTK_DIALOG(dialog));
             gtk_widget_destroy(dialog);
